src/components/TextArea/TextArea.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <component
+    :is="formElement ? CFormElement : CFragment"
+    v-bind="{ id, label, size, color, labelFormatter, message }"
```

Comment:
Need to pass stacked in too for FormElement
```suggestion
    v-bind="{ id, label, size, color, labelFormatter, message, stacked }"
```